samples/qa-a11y/src/views/EditableIntegerPicker.js

```diff
@@ -2,38 +2,24 @@ import EditableIntegerPicker from '@enact/moonstone/EditableIntegerPicker';
 import Heading from '@enact/moonstone/Heading';
 import Scroller from '@enact/moonstone/Scroller';
 import PropTypes from 'prop-types';
-import {Component} from 'react';
+import {useCallback, useState} from 'react';
 
-class EditableIntegerPickerWithAriaText extends Component {
-	static propTypes = {
-		defaultValue: PropTypes.number,
-		unit: PropTypes.string
-	};
+const EditableIntegerPickerWithAriaText = ({unit, ...props}) => {
+	const [value, setValue] = useState(0);
 
-	static defaultProps = {
-		defaultValue: 0,
-		unit: ''
-	};
+	const handleChange = useCallback((ev) => setValue(ev.value), []);
 
-	constructor (props) {
-		super(props);
-		this.state = {
-			value: props.defaultValue
-		};
-	}
+	const text = 'The current length is ' + value + unit;
 
-	handleChange = (ev) => this.setState({value: ev.value});
+	return (
+		<EditableIntegerPicker {...props} aria-valuetext={text} onChange={handleChange} />
```

Comment:
the first value is not correct and the unit is not displayed for any value. Can you check?
![image](https://user-images.githubusercontent.com/63335068/164440901-85ec4c0a-8229-43bb-aeea-6a7ae56ec860.png)

Reply:
Solved